project_report/tex/requirements.tex

```diff
@@ -9,7 +9,13 @@ \subsubsection{Functional \& Non-Functional Requirements}
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
 \subsubsection{The Simulator}
-
+The group are going to use the simulator software to test the software in the
+development cycle.
+The simulator is not a replacement of the machine since there is only
+so much randomness and correctness you can get out of a simulator, but it is 
```

Comment:
End sentence after "simulator"
Start new sentence with: "It is still very....."